libyul/optimiser/Suite.cpp

```diff
@@ -139,7 +140,10 @@ void OptimiserSuite::run(
 			_optimizeStackAllocation,
 			stackCompressorMaxIterations
 		);
-	suite.runSequence("fDnTOc g", ast);
+
+	// Run the user supplied (otherwise default) clean up sequence
```

Comment:
Well, from the point of view of this function it's always user-supplied :)
```suggestion
	// Run the user-supplied clean up sequence
```